hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -320,20 +320,22 @@ name:
         # V2
         "otbn_reset", "otbn_multi", "otbn_stress_all",
         "otbn_zero_state_err_urnd", "otbn_sw_errs_fatal_chk", "otbn_alert_test",
-        "otbn_intr_test", "otbn_tl_errors", "otbn_csr_hw_reset",
+        "otbn_intr_test", "otbn_tl_errors",
         "otbn_same_csr_outstanding"
         # V2 but known broken
         # "otbn_multi_err", "otbn_escalate",
         # V2S
         "otbn_imem_err", "otbn_dmem_err", "otbn_illegal_mem_acc",
         "otbn_tl_intg_err", "otbn_sec_cm", "otbn_pc_ctrl_flow_redun",
-        "otbn_rnd_sec_cm", "otbn_pc_ctrl_flow_redun", "otbn_alu_bignum_mod_err",
+        "otbn_rnd_sec_cm", "otbn_alu_bignum_mod_err",
         "otbn_controller_ispr_rdata_err", "otbn_mac_bignum_acc_err",
         # V2S but known broken
         # "otbn_passthru_mem_tl_intg_err",
         # V3 thus not yet active
         # "otbn_stress_all_with_rand_reset",
       ]
+      # Override reseed on these tests to run them only once.
+      reseed: 1
```

Comment:
I'd prefer to not override the absolute number of reseeds for all tests and instead run this regression suite with `--reseed-multiplier` < 1. That way, the number of reseeds per test can be controlled, but the number of reseeds of a test relative to an other is preserved. Some tests just need more reseeds than others to achieve reasonable coverage.

Ideally, a default value for `reseed-multiplier` could be set in the hjson, but I don't think that's possible?

Reply:
Okay, I have removed it. dvsim currently only supports overriding the `reseed` value in regressions.
However, I will recommend passing `--fixed-seed 1` in the command line instead of `--reseed-multiplier`. That fixes the seed value to 1 for all tests, which will make the CI less flaky. Otherwise, it will run with a random seed on every PR, and it may cause more frequent breakages which may have nothing to do with the PR.